react-calculator-app/src/components/Total.js

```diff
@@ -0,0 +1,8 @@
+import '../css/total.css';
+import { useCalculator } from '../store/CalcContext';
+const Total = () => {
+  const { ...state } = useCalculator();
+  return <h1 id="total">{state.total}</h1>;
```

Comment:
그냥 필요한 상태만 꺼내써도 되지 않을까요?
```suggestion
  const { total } = useCalculator();
  return <h1 id="total">{total}</h1>;
```

Reply:
[ff3d887](https://github.com/next-step/react-calculator/pull/15/commits/ff3d8878152bf2ff3889422721df5c6dafa66b6c) 에 반영하였습니다!